Classes/Recorders/LibObsRecorder.cs

```diff
@@ -106,6 +106,7 @@ public class LibObsRecorder : BaseRecorder {
 
             Connected = true;
             Logger.WriteLine("Successfully started LibObs!");
+            DetectionService.CheckAlreadyRunningPrograms();
```

Comment:
Call this in DetectionService.Start() or RecordingService.Start(), ideally LibObsRecorder doesn't even know that DetectionService exists